src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const ul = document.querySelector('.gallery__list');
+const div = document.querySelector('.gallery__large-img ');
+
+ul.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const link = e.target.closest('.list-item__link');
+
+  div.src = link.href;
```

Comment:
The 'div' variable is a reference to a DOM element, not an image element. Therefore, it does not have a 'src' property. You should use 'querySelector' to select an 'img' element inside the 'div' if that's your intention.